docs/REST.md

```diff
@@ -57,7 +58,8 @@ Retrieve an item.
         "item": {
             "address": "iGwEZUvfA",
             "modelAddress": "m8y7nEtAe",
-            "status": "AVAILABLE"
+            "status": "AVAILABLE",
+            "isCheckedOutTo": "student.id"
```

Comment:
Nit: Please use an actual student ID for the examples, e.g. `123456`.

Also, this should be a number and not a string, shouldn't it?